doc/flame/effects.md

```diff
@@ -328,21 +329,56 @@ final effect = AnchorToEffect(
 ```
 
 
-### `OpacityEffect`
+### `OpacityToEffect`
 
 This effect will change over time the opacity of the target to the specified alpha-value. Currently
 this effect can only be applied to components that have a `HasPaint` mixin. If the target component
 uses multiple paints, the effect can target any individual color using the `paintId` parameter.
 
+```{flutter-app}
+:sources: ../flame/examples
+:page: opacity_to_effect
+:show: widget code infobox
+:width: 180
+:height: 160
+```
+
 ```dart
-final effect = OpacityEffect.to(0.5, EffectController(duration: 0.75));
+ final effect = OpacityEffect.to(
+      0.2,
+      EffectController(duration: 0.75),
+    );
```

Comment:
None of these are correctly indented, put them into your IDE and run `flutter format` and you'll see.
It's supposed to be formatted just like the `ScaleEffect` examples:
```dart
final effect = ScaleEffect.by(
  Vector2.all(1.5),
  EffectController(duration: 0.3),
);
```

Reply:
Done✨